### DISCUSSION TITLE
Installation of Unique Bible on Windows 11

### DISCUSSION THREAD

**Discussion by onyaribo** [eliranwong/UniqueBible#860] [Q&A]
I ran into this error while trying to install.

_**"Selected binding "pyside2" could not be found, using "pyqt5""**_

The desktop app isn't working.  Have anyone experienced this and how did you solve it?

**Comment by otseng** [comment]
I have Windows 10 and it's working for me.

It could also be related to your version of Python.  What version of Python are you using?  Currently, the highest version supported is 3.9.

**Reply by Surcouf01** [reply]
I currently managed to make work UniqueBible with version 3.10.0.
With the latest version of UniqueBible I managed to make work uniquebible.exe on Windows.
Hi Oliver, how do I "release" this version ?

**Comment by Franstev99** [comment]
I have a near error with PyQt5 on Windows 11 and Python 3.9. it's correctly installed on c:\user... But when I type uba.py message ask to install PyQt5...

**Reply by Surcouf01** [reply]
Can you please try (having installed and selected python 3.10.0) :
pip install --upgrade uniquebible
And report exact step list and error message ?
Your description is a little bit vague.
